lib/nimble.phx.gen.template/variants/api/template.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Nimble.Phx.Gen.Template.Api.Template do
+  alias Nimble.Phx.Gen.Template.Project
+
+  def apply(%Project{} = project) do
+    project
```

Comment:
So nothing specific for API 😆

Reply:
For now yes @olivierobert since the current V1 template didn't have anything specific for API yet, maybe we could have some suggestions for API variant from GCP team. @rosle @bterone @hanam1ni